CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## v1.10.2
```

Comment:
I think the next version should be `1.11.0` as this is a breaking change, however you can remove this line and just add the list of things that is unreleased and I'll add the version when I release it.

Reply:
You're right !! I also dropped the line. 😉